src/app/customer/customer.component.html

```diff
@@ -0,0 +1,195 @@
+                         Please enter your name.
+                       </span>
+                       <span *ngIf="customerForm.get('Name')?.errors?.minlength">
+                         The name must be longer than 3 characters.
+                       </span>
+                     </span>
+             </div>
+           </div>
+
+
+           <div class="form-group">
+            <div class="col-md-12">
+                <label>Email</label>
+                <input type="text" 
+                       class="form-control"  
+                       formControlName="Email"  
+                       [ngClass]="{'is-invalid': (customerForm.get('Email')?.touched ||
+                                                  customerForm.get('Email')?.dirty) &&
+                                                 !customerForm.get('Email')?.valid }" />
+                      <span class="invalid-feedback">
+                       <span *ngIf="customerForm.get('Email')?.errors?.required">
+                         Please enter your email.
+                       </span>
+                       <span *ngIf="customerForm.get('Email')?.errors?.email">
+                        Please enter your email in correct format
+                       </span>
+                     </span>
+             </div>
+           </div>
+
+           <div class="form-group">
+
+            <div class="col-md-12">
+                <label >Location </label>
+                <br>
+                <div class="form-check-inline" *ngFor="let country of countries | async">
+                   <input class="form-check-input" 
+                          type="radio" 
+                          formControlName="Location"
+                          value="{{country}}"
+                          [ngClass]="{'is-invalid': (customerForm.get('Location')?.touched ||
+                                                     customerForm.get('Location')?.dirty) &&
+                                                    !customerForm.get('Location')?.valid }"
+                         > 
+
+                         <span class="invalid-feedback">
+                          <span *ngIf="customerForm.get('Location')?.errors?.required">
+                            Please select your location.
+                          </span>
+                          
+                        </span>
+                          {{ country }}
+                </div>
+                      <span class="invalid-feedback">
+                       <span *ngIf="customerForm.get('Name')?.errors?.required">
+                         Please enter your name.
+                       </span>
+                       <span *ngIf="customerForm.get('Name')?.errors?.minlength">
+                         The name must be longer than 3 characters.
+                       </span>
+                     </span>
+             </div>
+           </div>
+
+           <div class="form-group">
+            <label>Level</label>
+            <br>
```

Comment:
correct tag is <br/>

Reply:
Didn't get your point here. Can you please elaborate?